controllers/topics/index.js

```diff
@@ -341,15 +342,7 @@ const _topicAutoMessage = async (user_id, topic_id, detailTokenUser) => {
     });
 
     if (communityMessageFormat && !detailTokenUser.is_anonymous) {
-      await UserTopic.update(
-        {notify_user: true, is_anonymous: detailTokenUser.is_anonymous},
-        {where: {user_id, topic_id}}
-      );
-    } else {
-      await UserTopic.update(
-        {notify_user: false, is_anonymous: detailTokenUser.is_anonymous},
-        {where: {user_id, topic_id}}
-      );
+      followTopicServiceQueue(user_id, topic_id);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The replacement of the UserTopic update logic with a call to `followTopicServiceQueue` seems fine at first glance. However, it's crucial to ensure that the `followTopicServiceQueue` function handles all the necessary updates that were previously done by the UserTopic update logic. Also, make sure that the function properly handles any errors that might occur during the execution of the task in the queue.
```diff
-      await UserTopic.update(
-        {notify_user: true, is_anonymous: detailTokenUser.is_anonymous},
-        {where: {user_id, topic_id}}
-      );
-    } else {
-      await UserTopic.update(
-        {notify_user: false, is_anonymous: detailTokenUser.is_anonymous},
-        {where: {user_id, topic_id}}
-      );
-     }
+      followTopicServiceQueue(user_id, topic_id);
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->